src/routes/MemberProfile/MemberProfile.scss

```diff
@@ -213,7 +213,74 @@
   text-transform: uppercase;
   color: $color-rhino-30;
   font-size: $space-3x;
+  padding-top: $space-3x;
+  margin-top: $space-3x;
   margin-bottom: $space-3x;
+  border-top: solid 1px $color-rhino-10;
+}
+
+.profile-subhead:first-of-type {
+  margin-top: 0;
+  border-top: 0;
+}
+
+// PROJECTS
+
+.project {
+  background: #FFFFFF;
+  border: 1px solid #E1E5E9;
+  box-sizing: border-box;
+  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.05);
+  border-radius: 5px;
+  cursor: pointer;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  min-height: 60px;
+  padding: 8px 11px;
+  margin: 7px auto;
+
+    .title {
```

Comment:
I'm low-key wishing the indentation was the standard (in Hylo stuff so far) 2 spaces here instead of 4. Because standards, nothing more.